application/src/main/java/org/togetherjava/tjbot/commands/free/UserStrings.java

```diff
@@ -12,6 +12,8 @@
             Thank you for asking in an available channel. A helper will be with you shortly!
 
             __Do not post your question in other channels.__
+                 
+            After you are satisfied with the given answer don't forget to free the channel for others with the '/free' command!
```

Comment:
Imo this sounds kinda harsh, with the exclamation mark and the "dont foget to".

I would rephrase it and shorten to:

> Once your issue is resolved, feel free to `/free` the channel, thanks.

Reply:
+1